### ISSUE TITLE
Nesidioblastosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7669]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
An abnormality of pancreatic beta cells characterized by  a proliferation of abnormal beta cells throughout the entire pancreas, with enlarged islet size and number (hypertrophic islets), increased periductular islets, enlarged beta-cell nuclei and abundant clear cytoplasm. Occasionally beta cells with pleomorphic nuclei, ductuloinsular complexes, and neoformation of islets from ducts are observed.
PMID:24645840
PMID:15517495

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
601820 

PMID: 8923010 1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
